torch/nn/utils/rnn.py

```diff
@@ -143,9 +143,7 @@ def pack_padded_sequence(input, lengths, batch_first=False):
     return PackedSequence(data, batch_sizes)
 
 
-def _symbolic_pack_padded_sequence(g, input, lengths, batch_first=False, padding_value=0.0, total_length=None):
-    if total_length is not None:
```

Comment:
just to double-check, this change was made in fbcode and you approve of it @bddppq ?

Reply:
This was made in fbcode and I didn't approve it but it does look reasonable to me. @anderspapitto for double check.

Reply:
I reviewed/approved this in fbcode